widgets/var_widgets.py

```diff
@@ -272,13 +272,10 @@ def insert(self, *a, **kw):
         to_track = []
 
         try:
-            values = kw["values"]
+	        kw["values"] = values = list(kw.pop("values"))
```

Comment:
Sorry. The tab was wrapped in code snippet.
https://github.com/ispras/qdt/pull/38#discussion_r386996132 - I ran the GUI and received no exceptions.